### PR TITLE
Add TESTING document and simplify README

### PR DESCRIPTION
**Pull request by MikeMcC399**
This PR implements the suggestion from https://github.com/corona-warn-app/cwa-website/issues/3410.

Detailed testing information is moved from [README: Testing](https://github.com/corona-warn-app/cwa-website/blob/master/README.md#testing) into a new docs/TESTING.md file, which has re-worked contents that reflect the introduction of automatic testing. Some text from the original README which is no longer necessary has been removed.

Whilst this PR is open and not yet merged, the documents can be viewed under:

- [docs/testing-update/README.md](https://github.com/MikeMcC399/cwa-website/blob/docs/testing-update/README.md)
- [docs/TESTING.md](https://github.com/MikeMcC399/cwa-website/blob/docs/testing-update/docs/TESTING.md)

### CODE REVIEW

**Review by brianebeling — APPROVED** [review]
Thank you very much for this PR. The new additions like `npx cypress cache prune` are very much appreciated. 
Testes every command, works.

**Comment by MikeMcC399** [comment]
@brianebeling 

Thanks for checking! Perhaps leave the PR for a day or two before merging to see if there are any other comments or points which are unclear?

The Cypress cache description was previously in the README. I moved it to TESTING and shortened it.

**Comment by brianebeling** [comment]
> Brian-J. Ebeling
> 
> Thanks for checking! Perhaps leave the PR for a day or two before merging to see if there are any other comments or points which are unclear?
> 
> The Cypress cache description was previously in the README. I moved it to TESTING and shortened it.

Sure. Additions may have been the wrong term - rewordings. I think it reads as good as new.

File: README.md
Comment (larswmh) [inline README.md:80]:
```suggestion
Most tests run automatically when a pull request is submitted. If you are making changes which affect hyperlinks in the web it is recommended to run the links test locally before submitting a pull request or making changes to it.
```

If you are making changing -> If you are making changes

Comment (MikeMcC399) [reply README.md]: Thanks for noticing the typo!

**Review by larswmh — APPROVED** [review]

File: docs/TESTING.md
Comment (Ein-Tim) [inline docs/TESTING.md:74]:
```suggestion
| hotline_spec       | Verify that app hotline is correctly displayed on the home page.                                                        |
```

1. Wording change
2. Isn't this check for the footer and not the home page?

File: docs/TESTING.md
Comment (Ein-Tim) [inline docs/TESTING.md:94]: 👍 Very good idea to mention that!

**Review by Ein-Tim — COMMENTED** [review]
Looks very good in general, thank you!

File: docs/TESTING.md
Comment (MikeMcC399) [reply docs/TESTING.md:74]: @Ein-Tim 

It's the footer on every page, including the "home page" and it displays the literal text "Hotline App", so I believe this is correct.

image

Comment (Ein-Tim) [reply docs/TESTING.md:74]: @MikeMcC399 Right, thanks, that's the text! Maybe change it from "home page" to footer, than? But I'll leave that to you!

Comment (Ein-Tim) [reply docs/TESTING.md:74]: Ok, thanks

**Review by Ein-Tim — APPROVED** [review]
LGTM!

**Pull request merged**
